editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/CamelEditorIT.java

```diff
@@ -307,6 +308,85 @@ public void dropRouteOnRoute() throws Exception {
 		assertThat(createRouteFigureFeature.canExecute(createCtx)).isFalse();
 	}
 	
+	@Test
+	public void reconnectToUnwiredNode() throws Exception {

```

Comment:
better to provide more explicit test name: something like: reconnectToUniredNodeShouldBePossible